cmd/prysmctl/validator/cmd.go

```diff
@@ -20,22 +20,50 @@ var (
 		Usage: "host:port for beacon node to query",
 		Value: "127.0.0.1:3500",
 	}
+
 	PathFlag = &cli.StringFlag{
 		Name:    "path",
 		Aliases: []string{"p"},
 		Usage:   "path to the signed withdrawal messages JSON",
 	}
+
 	ConfirmFlag = &cli.BoolFlag{
 		Name:    "confirm",
 		Aliases: []string{"c"},
 		Usage: "WARNING: User confirms and accepts responsibility of all input data provided and actions for setting their withdrawal address for their validator key. " +
 			"This action is not reversible and withdrawal addresses can not be changed once set.",
 	}
+
 	VerifyOnlyFlag = &cli.BoolFlag{
 		Name:    "verify-only",
 		Aliases: []string{"vo"},
 		Usage:   "overrides withdrawal command to only verify whether requests are in the pool and does not submit withdrawal requests",
 	}
+
+	ValidatorHostFlag = &cli.StringFlag{
+		Name:    "validator-client-host",
```

Comment:
There isn't a client/server split to differentiate with the term "validator", so I think just calling it the "validator" would be clear and concise vs "validator-client".